ui/segmented-bar.reel/segment.reel/segment.css

```diff
@@ -0,0 +1,13 @@
+/* horizontal */
+.montage-Segment {
+    height: 100%;
+    box-sizing: border-box;
+    border: 1px solid black;
+    background: darkgray;
+}
+
+/* vertical, not used, set in JS */
+/*@media (max-width: 1000px) {*/
+/*.montage-Segment {*/
+/*}*/
+/*}*/
```

Comment:
Sorry should say "TBD, currently not used", b/c we need to decide whether to use media queries in CSS or JS.